script.js

```diff
@@ -0,0 +1,111 @@
+let calcDisplay = document.querySelector("#calc-display");
+let butID = document.querySelectorAll("button")
+let prevNum = "";
+let nextNum = 0;
+let operatorsVal = "+";
+
+function display(){
+nextNum = this.getAttribute('id');
+switch (nextNum) {
+    case '1':
+    nextNum = 1;
+    numSequence();
+    break;
+
+    case '2':
+    nextNum = 2;
+    numSequence();
+    break;
+    
+    case '3':
+    nextNum = 3;
+    numSequence();
+    break;
+
+    case '4':
+    nextNum = 4;
+    numSequence();
+    break;
+
+    case '5':
+    nextNum = 5;
+    numSequence();
+    break;
+
+    case '6':
+    nextNum = 6;
+    numSequence();
+    break;
+
+    case '7':
+    nextNum = 7;
+    numSequence();
+    break;
+
+    case '8':
+    nextNum = 8;
+    numSequence();
+    break;
+
+    case '9':
+    nextNum = 9;
+    numSequence();
+    break;
+
+    case '0':
+    nextNum = 0;
+    numSequence();
+    break;       
+}        
+function numSequence(){
+    prevNum = `${prevNum}${nextNum}`;
+    calcDisplay.innerHTML = prevNum;
+}
+}
+
+function equals(){
+    calcDisplay.innerHTML = eval(prevNum);

```

Comment:
avopid using eval in all possible cases, it is considered a huge security risk since it b ascially computes raw strings, which can be malicious. 

if you case, since you are essentially building up strings like '11 + 11' and then tryign to figure out how to compute it, based on your approach, when you click an operator, keep trrack of it. then you can take your string and split it at the operator. 

eg: 

`let myResultString = '111 + 222' 
 const numberts = myResultString.split(myStoredOperatorString)  // ['111' , '222']`
 
 the idea is that since you know what the operator is, you can easily split your string and then you will always have 2 numbers in the split array arr[0] which is left side of equation and arr[1] which is the right side of the equation, you can then convert to a number and run the operation